src/main/java/gregtech/api/unification/ore/OrePrefix.java

```diff
@@ -244,7 +244,7 @@ public static class Conditions {
     }
 
     static {
-        ingotHot.heatDamage = 3.0F;
+        ingotHot.heatDamageFunction = (temp) -> ((temp - 1750) / 500.0F) + 3;
```

Comment:
Do we want to scale this in half steps, so 3 damage, 3.5 damage, 4 damage for every 500 degrees above the min hot ingot temp, or is scaling by 1 damage for every 500 heat ok damage wise?